static/css/mozsearch.css

```diff
@@ -619,9 +619,20 @@ tr.after-context-line + tr.before-context-line {
 .panel a.field {
   background-position: 2px 10px;
 }
-.panel section a {
+.panel section .item {
   display: inline-block;
   width: 100%;
+  cursor: pointer;
+}
+.panel section button.item {
+  font: unset;
+  appearance: unset;
+  border: unset;
+  text-align: unset;
+}
+.panel section button.item[disabled] {
```

Comment:
nit: you can use `.panel section .item:disabled` probably, it's a bit more to the point.